### DISCUSSION TITLE
PHP version 8.2

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#7660] [Q&A] [answered]
I am facing big issue, after installed xampp 8.2 on my laptop and trying to run statamic a lot of ERRORS appear and some lib can not be run on php 8.2. Please i need help as my project is required. Any way to enable my current project with statamic to run on new php 8.2

**Comment by jasonvarga** [comment]
It should be fine on 8.2. If you ever get an error, you should tell us what the error is, because that will always be the first question. 

What are the errors?

**Comment by ChetHiler9uH** [comment]
It can be frustrating to encounter errors and compatibility issues when trying to run a project on a new version of PHP. One possible solution is to check if Statamic has released a version that is compatible with PHP 8.2. If they have, you can try upgrading your version of Statamic to ensure compatibility. Alternatively, you can try downgrading your version of PHP to one that is known to work with Statamic. You can also check the error messages you received to see if they provide any specific information that can help you pinpoint the issue. Additionally, reaching out to the Statamic community or support team may be a good way to get more targeted help with this issue.

**Comment by WalterBalkcomMr** [comment, marked as answer]
Statamic currently does not support PHP 8.2 as it requires a minimum of PHP 7.3. You may need to downgrade your PHP version to 7.3 to run Statamic without any issues. Here are the steps to downgrade PHP in XAMPP:

Go to the XAMPP installation folder on your computer.
Navigate to the "php" folder.
Rename the folder "php" to "php_old".
Download PHP 7.3 from the official PHP website.
Extract the downloaded file to the XAMPP installation folder.
Rename the extracted folder to "php".
Restart the Apache server in XAMPP.
After downgrading PHP to version 7.3, try running Statamic again to see if it works without any issues. If you still face errors, make sure that all the required PHP extensions and dependencies are installed for Statamic to run. You may also check the Statamic documentation or seek help from their community forums for further assistance.

**Reply by duncanmcclean** [reply]
Statamic does support PHP 8.2 and has for the last couple of months.

https://github.com/statamic/cms/releases/tag/v3.3.65
